doc/model_details/HillTononi.ipynb

```diff
@@ -0,0 +1,792 @@
+    "\\end{equation}\n",
+    "\n",
+    "The neuron emits a single spike if \n",
+    "- it is not refractory\n",
+    "- membrane potential crosses the threshold, $V\\geq\\theta$\n",
+    "\n",
+    "Upon spike emission,\n",
+    "- $V \\leftarrow E_{\\text{Na}}$\n",
+    "- $\\theta \\leftarrow E_{\\text{Na}}$\n",
+    "- the neuron becomes refractory for time $t_{\\text{spike}}$ (`t_ref` in NEST)\n",
+    "\n",
+    "The repolarizing current is active during, and only during the refractory period:\n",
+    "\\begin{equation}\n",
+    "g_{\\text{spike}} = \\begin{cases}  1  & \\text{neuron is refractory}\\\\\n",
+    " 0 & \\text{else} \\end{cases}\n",
+    "\\end{equation}\n",
+    "\n",
+    "During the refractory period, the neuron cannot fire new spikes, but all state variables evolve freely, nothing is clamped. \n",
+    "\n",
+    "The model of spiking and refractoriness is based on Synthesis model `PulseIntegrateAndFire`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Intrinsic currents\n",
+    "\n",
+    "Note that not all intrinsic currents are active in all populations of the network model presented in [HT05, p1678f].\n",
+    "\n",
+    "Intrinsic currents are based on the Hodgkin-Huxley description, i.e.,\n",
+    "\n",
+    "\\begin{align}\n",
+    "I_X &= g_{\\text{peak}, X} m_X(V, t)^N_X h_X(V, t)(V-E_X) \\\\\n",
+    "\\frac{\\text{d}m_X}{\\text{d}t} &= \\frac{m_X^{\\infty}-m_X}{\\tau_{m,X}(V)}\\\\\n",
+    "\\frac{\\text{d}h_X}{\\text{d}t} &= \\frac{h_X^{\\infty}-h_X}{\\tau_{h,X}(V)}\n",
+    "\\end{align}\n",
+    "\n",
+    "#### Pacemaker current $I_h$\n",
+    "\n",
+    "Synthesis: `IhChannel`\n",
+    "\n",
+    "\\begin{align}\n",
+    "N_h & = 1 \\\\\n",
+    "m_h^{\\infty}(V) &= \\frac{1}{1+\\exp\\left(\\frac{V+75\\text{mV}}{5.5\\text{mV}}\\right)} \\\\\n",
+    "\\tau_{m,h}(V) &= \\frac{1}{\\exp(-14.59-0.086V) + \\exp(-1.87  + 0.0701V)} \\\\\n",
+    "h_h(V, t) &\\equiv 1 \n",
+    "\\end{align}\n",
+    "\n",
+    "Note that subscript $h$ in some cases above marks the $I_h$ channel, \n",
+    "\n",
+    "#### Low-threshold calcium current $I_T$\n",
+    "\n",
+    "Synthesis: `ItChannel`\n",
+    "\n",
+    "##### Equations given in paper \n",
+    "\n",
+    "\\begin{align}\n",
+    "N_T & \\quad \\text{not given} \\\\\n",
+    "m_T^{\\infty}(V) &= 1/\\{1 +  \\exp[ -(V +  59.0)/6.2]\\} \\\\\n",
+    "\\tau_{m,T}(V) &= \\{0.22/\\exp[ -(V  + 132.0)/ 16.7]\\} +  \\exp[(V  + 16.8)/18.2] +  0.13\\\\\n",
+    "h_T^{\\infty}(V) &= 1/\\{1 +  \\exp[(V +  83.0)/4.0]\\} \\\\\n",
+    "\\tau_{h,T}(V) &= \\langle  8.2 +  \\{56.6 +  0.27 \\exp[(V +  115.2)/5.0]\\}\\rangle / \\{1.0 +  \\exp[(V +  86.0)/3.2]\\}\n",
+    "\\end{align}\n",
+    "\n",
+    "Note the following:\n",
+    "- $N_T=2$ from Synthesis code\n",
+    "- In the equation for $\\tau_{m,T}$, the second exponential term must be added to the first (in the denominator) to make dimensional sense\n",
```

Comment:
I guess this means that the constants 0.22 and 0.13 have the dimension of a time? could you precise this?
EDIT: ok, this appears just below... but still ^^